consensus/src/dag/dag_store.rs

```diff
@@ -296,4 +359,43 @@ impl Dag {
 
         DagSnapshotBitmask::new(lowest_round, bitmask)
     }
+
+    pub(super) fn prune(&mut self) {
```

Comment:
Wen GC?

Reply:
we are pruning after state sync finishes so we only have the latest DAG.